ports/azure_sdk.md

```diff
@@ -0,0 +1,341 @@
+
+ - Note: You will not need all of the ports provided for the `ia32-generic-qemu` target architecture, these ports are necessary: `busybox`, `openssl`, `curl` and `azure_sdk` of course, more about building script You can find [here](https://github.com/phoenix-rtos/phoenix-rtos-doc/blob/master/building/script.md)
+
+
+Next, You will need an internet connection. Here is an example of how to setup the network on `ia32-generic-qemu`:
+
+### Network setup on ia32-generic-qemu
+
+- Note: This guide was tested on `Ubuntu 20.04 LTS` host OS.
+
+There are a few steps to follow:
+
+ 1. Create and set up `vibr0` bridge on a host using `qemu-bridge-helper`:
+
+  - Install the required package and ensure that `libvirtd` is running:
+
+  ```
+  sudo apt-get update
+  sudo apt-get install qemu-system-common
+  systemctl enable libvirtd.service
+  systemctl start libvirtd.service
+  ```
+
+  - Start the default network bridge, and configure it to run on startup.
+
+  ```
+  sudo virsh net-autostart --network default
+  sudo virsh net-start --network default
+  ```
+
+  - After that verify that the IP range `192.168.122.1/24` is reported by the `vibr0` bridge:
+
+  ```
+  ip addr show virbr0
+  ```
+
+  - The expected output:
+
+  ```
+   virbr0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default qlen 1000
+      link/ether xx:xx:xx:xx:xx:xx brd ff:ff:ff:ff:ff:ff
+      inet 192.168.122.1/24 brd 192.168.122.255 scope global virbr0
+         valid_lft forever preferred_lft forever
+  ```
+
+  - Set up `qemu-bridge-helper` (`chmod` is used here to allow running qemu without root privileges)
+
+  ```
+  echo "allow virbr0" > /etc/qemu/bridge.conf
+  sudo chmod a+rw /etc/qemu/bridge.conf
+  ```
+
+  - The `/etc/qemu` directory may not exist, then create it and provide the required privileges:
+
+  ```
+  sudo mkdir /etc/qemu
+  sudo chmod a+rw /etc/qemu
+  echo "allow virbr0" > /etc/qemu/bridge.conf
+  sudo chmod a+rw /etc/qemu/bridge.conf
+  ```
+
+  <img src="_images/azure_sdk_vibr_setup.png" width="700px">
+
+  - Sources: https://apiraino.github.io/qemu-bridge-networking/, https://mike42.me/blog/2019-08-how-to-use-the-qemu-bridge-helper-on-debian-10
+
+ 2. Launch `qemu` using a starting script with `net` suffix:
+
+  ```
+  ./scripts/ia32-generic-qemu-net.sh
+  ```
+
+ 3. Configure network and run `ash` (Busybox applet) using `rc` script:
+
+  - Note: `IP` is assigned using `DHCP`, there are also other possibilities, for more please check the configuration file located in `_projects/ia32-generic-qemu/rootfs-overlay/etc/rc.conf.d/network`
+
+  - Note: There are also executed other programs, for more information please check the content of the `rc` file for `ia32-generic-qemu` in `_projects/ia32-generic-qemu/rootfs-overlay/etc/rc`
+
+  ```
+  /linuxrc
+  ```
+
+  - As You can see, the advanced version of `Phoenix-RTOS` with `POSIX` shell has been started:
```

Comment:
```suggestion
  - As you can see, the advanced version of `Phoenix-RTOS` with `POSIX` shell has been started:
```
I also don't know, what you mean by `advanced`. You might have meant `expanded` or `extended`?